web/src/test/java/org/cbioportal/web/StudyViewControllerTest.java

```diff
@@ -5,24 +5,10 @@
 import java.util.Collections;
 import java.util.List;
 
-import org.cbioportal.model.ClinicalAttribute;
-import org.cbioportal.model.ClinicalData;
-import org.cbioportal.model.ClinicalDataCount;
-import org.cbioportal.model.ClinicalDataCountItem;
-import org.cbioportal.model.CopyNumberCountByGene;
-import org.cbioportal.model.GenePanelData;
-import org.cbioportal.model.MolecularProfile;
-import org.cbioportal.model.MutationCountByGene;
-import org.cbioportal.model.Sample;
-import org.cbioportal.service.ClinicalAttributeService;
-import org.cbioportal.service.ClinicalDataService;
-import org.cbioportal.service.DiscreteCopyNumberService;
-import org.cbioportal.service.GenePanelService;
-import org.cbioportal.service.MolecularProfileService;
-import org.cbioportal.service.MutationService;
-import org.cbioportal.service.PatientService;
-import org.cbioportal.service.SampleService;
-import org.cbioportal.service.TreatmentService;
+import org.cbioportal.model.*;
```

Comment:
I think we adopted the google java style guidelines, which forbid wildcards with import statements.
(https://google.github.io/styleguide/javaguide.html#s3.3-import-statements)
That's overly restricted .. and I think use of them with built in packages, such as import java.util.* is fine ... but maybe for our own defined classes (which sometimes do have naming clashes across packages) we should be more explicit, showing the individual classes imported. It also can be helpful in terms of documentation ... to see what is connected to the current class. I think there was a similar collapse of web.parameter.* up higher in the code. I think I would prefer if we keep these details for our own packages (even if it is verbose)

Reply:
Ah ok, this just happens when I automatically organize imports with IntelliJ. I have removed all wildcards from imports.

Reply:
:+1: